glossary.md

```diff
@@ -87,7 +87,16 @@ Property of [variables](#variable) that measure proportions, used in conjunction
 
 As an example, in 1999, [approximately 36% of Canadians were Internet users](https://datacommons.org/browser/dc/o/0d9e3dd3y6yt3){: target="_blank"}. Here the measured value of `Count_Person_IsInternetUser_PerCapita` is 36, and the scaling factor or denominator for this per capita measurement is 100. Without the scaling factor, we would interpret the value to be 36/1, or 3600%.
 
-A complete list of properties can be found in the [Knowledge Graph](https://datacommons.org/browser/scalingFactor){: target="_blank"}.
+### [Statistical Variable](https://datacommons.org/browser/StatisticalVariable){: target="_blank"}
+{: #variable}
+
+Any type of metric, statistic, or measure that can be measured at a place and time. Examples include [median income of persons older than 16](https://datacommons.org/browser/Median_Income_Person_16OrMoreYears){: target="_blank"}, [number of female high school graduates aged 18 to 24](https://datacommons.org/browser/Count_Person_18To24Years_EducationalAttainmentHighSchoolGraduateIncludesEquivalency_Female){: target="_blank"}, [unemployment rate](https://browser.datacommons.org/browser/UnemploymentRate_Person){: target="_blank"}, or [percentage of persons with diabetes](https://browser.datacommons.org/browser/Percent_Person_WithDiabetes){: target="_blank"}. A complete list of variables can be found in the [Knowledge Graph](https://datacommons.org/browser/StatisticalVariable){: target="_blank"}.
```

Comment:
So it's typically associated with a place and time but in the more generic sense, it can be associated with any entity in the graph (e.g. business unit, power plant, etc.) and time.

Reply:
So I actually didn't write or update this, I only put it in alphabetical order. :-) But updated as you suggest.